docs/guides/overview.md

```diff
@@ -0,0 +1,188 @@
+# Using Radium
+
+Radium is a toolset for easily writing React component styles. It resolves modifiers, browser states, and media queries to apply the correct styles to your components, all without selectors, specificity, or source order dependence.
+
+## How do I do it, then?
+
+Let's create a fictional `<Button>` component. It will have a set of default styles, will adjust its appearance based on modifiers, and will include hover, focus, and active states.
+
+```js
+var Button = React.createClass({
+  render: function () {
+    return (
+      <button>
+        {this.props.children}
+      </button>
+    );
+  }
+});
+```
+
+Most of Radium's behavior is included in mixins that provide different types of functionality. For basic use, you'll want to start with the `StyleResolverMixin`. This mixin builds out the styles that you'll apply to your component.
+
+```js
+var Button = React.createClass({
+  mixins: [ StyleResolverMixin ],
+
+  render: function () { ... }
+});
+```
+
+`StyleResolverMixin` has a method called `buildStyles()` that converts a nested Radium style object into the styles that should be applied to your component based on its props and state. A basic Radium style object looks like this:
+
+```js
+{
+  standard: {
```

Comment:
Could further change this code block to: `var radStyles = ...` so that the example you show below is valid vs "where did `radStyles` come from, is it special?"